packages/whatsie/whatsie.pacscript

```diff
@@ -0,0 +1,19 @@
+name="whatsie"
+gives="whatsie"
+version="4.14.1"
+url="https://github.com/keshavbhatt/whatsie/archive/refs/tags/v${version}.tar.gz"
+hash="369acebb948375713f4e97a5e0304dbda407b432b42fe8ad0b6aa9b602744311"
+homepage="https://github.com/keshavbhatt/whatsie"
+description="Feature rich WhatsApp Client for Desktop Linux"
+maintainer="Diegiwg <diegiwg@gmail.com>"
+depends=("libqt5webenginewidgets5")
```

Comment:
```suggestion
depends=("libqt5webengine5" "libqt5webenginewidgets5" "libqt5location5")
```
These are some of the main QT libraries the app is using, these packages will pull other required runtime dependencies So, not mentioning them explicitly.

Reply:
I can't say if it was those two packages, but after adding them as dependencies, we started to get this error, in Debian-Unstable:

"keyboard-configuration : Depends: xkb-data (< 2.35.1A) but 2.38-2 is to be installed"

Reply:
That's an issue with debian sid right now. Don't worry about that.

Reply:
Good, then it's all right!